images/dev-api-server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG KAMU_VERSION
```

Comment:
super nitpick:

CLI image lives in the directory `images/kamu-base-dev` so mb for consistency let's use `image/api-server-dev` too?

I prefer things that are closely related to be close to each other lexicographically, so suffixing is better.

PS: I know that we also chose prefixing for actual image names (`kamu-dev-api-server`) but I see this as top-level separation of dev vs prod images.

Reply:
Totally reasonable, thanks for the comment

Reply:
> CLI image lives in the directory images/kamu-base-dev

I changed this aspect in https://github.com/kamu-data/kamu-cli/pull/969

![image](https://github.com/user-attachments/assets/73a5f467-e38b-4e45-8f22-4e835ec6b0d5)

--

Hm, I'll think about it for a while before choosing (again). I was also in a dilemma: matching the image name VS naming folders for visualization purposes...

Reply:
Since this is about consistency, I added the `kamu-` prefix for images in this repository, just as we did with `kamu-cli`.